src/org/usfirst/frc/team2175/SolenoidWrapper.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc.team2175;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.Solenoid;
+
+public class SolenoidWrapper {
+    private DoubleSolenoid doubleSolenoid;
+    private Solenoid solenoid;
+    private boolean isDouble;
+
+    public SolenoidWrapper(final String[] solenoidInfo) {
```

Comment:
@4thwind Why don't you make a `SolenoidInfo` class, like we did for `ButtonInfo`?